keyboards/keebio/iris/keymaps/khitsule/keymap.c

```diff
@@ -0,0 +1,154 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _GAME 1
+#define _LOWER 2
+#define _RAISE 3
+#define _ADJUST 4
+
+// define keycode macros for readability
+#define KM_DLEFT LGUI(LCTL(KC_LEFT))
+#define KM_DRIGHT LGUI(LCTL(KC_RIGHT))
+#define TSKMGR LCTL(LSFT(KC_ESC))
+#define _______ KC_TRNS
```

Comment:
```suggestion
#define _______ KC_TRNS
```

Reply:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```